app/_templates/layout.njk

```diff
@@ -9,7 +9,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
 
-    <title>NHS.UK frontend - {{ title }}</title>
+    <title>{{ title }} | NHS.UK frontend</title>
```

Comment:
This should probably be 'Page title - NHS.UK frontend' to match the guidance in the service manual ([https://beta.nhs.uk/service-manual/accessibility/development#set-page-titles](https://beta.nhs.uk/service-manual/accessibility/development#set-page-titles))